2-mandatory/4-restaurant.js

```diff
@@ -21,7 +21,24 @@ const MENU = {
 
 let cashRegister = {
   // write code here
-}
+  orderBurger: function (balance) {
+    if (balance >= MENU.burger) {
+      let newBalance = balance - MENU.burger;
+      return newBalance;
+    } else {
+      return balance;
+    }
+  },
```

Comment:
good job here, this is a very clear solution & your variable names make it easy to understand if you are unfamiliar with the code